commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/action/CPSearchResultsConfigurationAction.java

```diff
@@ -48,8 +48,8 @@
 
 	@Override
 	public String getJspPath(HttpServletRequest httpServletRequest) {
-		CPRequestHelper cpRequestHelper = new CPRequestHelper(
-			httpServletRequest);
+		CPSearchResultsRequestHelper cpRequestHelper =
```

Comment:
`CPSearchResultsRequestHelper cpRequestHelper` should be `CPSearchResultsRequestHelper cpSearchResultsRequestHelper`